modules/02_star.nf

```diff
@@ -13,12 +13,12 @@ process STAR {
 
     output:
       tuple val("${name}"), file("${name}.bam"), emit: bams
-      file("software_versions.${task.process}.txt")
+      file("software_versions.STAR.txt")
 
     script:
     two_pass_mode_param = params.star_two_pass_mode ? "--twopassMode Basic" : ""
     """
-    STAR --genomeDir ${params.reference} ${two_pass_mode_param} ${params.additional_args} \
+    STAR --genomeDir ${reference} ${two_pass_mode_param} ${params.additional_args} \
```

Comment:
you are missing the definition of input parameter `val(reference)`

Reply:
Added missing input parameter in latest commit